src/hooks/useStore.js

```diff
@@ -1,16 +1,14 @@
 import create from 'zustand';
 
-const useStore = create(set => ({
-	counter: 0,
-	setCounter(counter) {
-		set({counter});
-	},
-	decrementCounter(step = 1) {
-		set(({counter}) => ({counter: counter - step}));
-	},
-	incrementCounter(step = 1) {
-		set(({counter}) => ({counter: counter + step}));
-	},
-}));
+const useStore = create(set => {
+	return {
+		entries: [],
+		addEntry: name => {
+			set(state => {
+				return {entries: [...state.entries, {name}]};
+			});
+		},
+	};
+});
```

Comment:
You are not using it at the moment. If you want to implement Zustand later on, then remove it for now and add it later again :)